example/declarative.dart

```diff
@@ -1,29 +1,31 @@
 // Copyright 2013 Google. All rights reserved. Use of this source code is
 // governed by a BSD-style license that can be found in the LICENSE file.
 
-library grinder_example_1;
+library grinder.example.declarative;
 
 import 'package:grinder/grinder.dart';
 
-void main([List<String> args]) {
-  task('init', init);
-  task('compile', compile, ['init']);
-  task('deploy', deploy, ['compile']);
-  task('default', null, ['deploy']);
-
+void main(List<String> args) {
+  addAnnotatedTasks();
```

Comment:
That would certainly be terse :) I know that's a pattern people have used - I've seen it in dart polymer. For me it seems like depending on a side effect - that import (or exporting) a package causes code to be run. And the flow for the user or reader isn't as clear as having an explicit main() method.

The explicit main() is a bit of boilerplate however. So, I'd be ok if you wanted to add a `package:grinder/main.dart` library, I just would want to make it clear that both implicit and explicit ways of starting the library were OK.